### ISSUE TITLE
Complain when pkg.smacspec references version without tag

### ISSUE THREAD

**Issue by ratmice** [standardml/smackage#27]
I inevitably forget to add a tag when bumping pkg.smacspec's version,
This seems to currently silently ignore the update,
it would be nice if it complained, or perhaps just a verbose mode to help figuring out why.

**Comment by gian** [comment]
Good idea. I can probably take a look at this in the next couple days.
